src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -113,16 +123,14 @@ export function DAppsExplorerScreen() {
         {t('dappsScreenHelpDialog.message')}
       </Dialog>
       <>
-        {loading && !categoriesById && (
-          <View style={styles.centerContainer}>
-            <ActivityIndicator
-              style={styles.loadingIcon}
-              size="large"
-              color={colors.greenBrand}
-              testID="DAppExplorerScreen/loading"
-            />
-          </View>
-        )}
+        <Animated.View style={[styles.loadingContainer, animatedStyle]}>
+          <ActivityIndicator
+            size="large"
+            color={colors.greenBrand}
+            testID="DAppExplorerScreen/loading"
+          />
+        </Animated.View>
+
```

Comment:
nit: Seeing the animation made me think the screen would support pull-to-refresh, when it doesn't 😅 
Any issue turning this as the [refreshControl](https://reactnative.dev/docs/refreshcontrol) of the ScrollView instead?